packaging/opensuse/snapd.spec

```diff
@@ -14,6 +14,9 @@
 
 # Please submit bugfixes or comments via http://bugs.opensuse.org/
 
+# takes an absolute path with slashes and turns it into an AppArmor profile path
+%define as_apparmor_path() %(echo "%1" | tr / . | cut -c2-)
```

Comment:
I didn't know RPM had functions like that.

Reply:
RPM macro language is a mash of m4 and shell, so it has properties from both.